scripts/results/demos/plot_fanning.py

```diff
@@ -54,20 +54,24 @@
     os.mkdir('figures')
 
 
-def adjust_fig(fig, include_cbar=False):
-    fig.set_size_inches(2.5, 2.0, forward=True)
+def adjust_fig(fig, label, title, include_cbar=False):
```

Comment:
I just noticed this. I feel we should try to keep matplotlib kungfu to the minimum or hide it in the config.py file. Even though it looks a little more polished for publications, we're increasing the entry barrier to reuse these scripts. But let's not worry about it now. We can fix it later.